app/Console/Commands/DeployInitial.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+
+class DeployInitial extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'deploy:initial';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'All commands that should be run when a container boots.';
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        // split migrate commands between production and development versions
+        if (config('app.env') === 'production') {
+            $this->call('migrate', ['--step' => true, '--force' => true]);
+        } else {
+            $this->call('db:refresh');
```

Comment:
This won't do anything in testing because of the conditional check here: https://github.com/accessibility-exchange/platform/blob/ca5f89b0803377ce491c771b771cb5f138ba9057/app/Console/Commands/DatabaseRefresh.php#L31

Reply:
Should we remove these tests altogether?

Reply:
I think so @marvinroman.